frontend-react/src/pages/misc/FeatureFlags.tsx

```diff
@@ -0,0 +1,150 @@
+import { Helmet } from "react-helmet";
+import { useRef, useState } from "react";
+import {
+    Alert,
+    Button,
+    Grid,
+    GridContainer,
+    Label,
+    TextInput,
+} from "@trussworks/react-uswds";
+
+import { showAlertNotification } from "../../components/AlertNotifications";
+
+/* feature flags are just and array of strings saved into a single localStorage variable */
+const LOCALSTORAGE_KEY = "featureFlags";
+
+function getSavedFeatureFlags(): string[] {
+    const saved = window.localStorage.getItem(LOCALSTORAGE_KEY) || "";
+    if (saved === "") {
+        return [];
+    }
+    return saved.split("\t");
+}
+
+function addFeatureFlag(flag: string) {
+    const flagClean = flag.trim().toLowerCase();
+    if (flagClean === "") {
+        return;
+    }
+
+    // new Set() //   => remove duplication
+    const set = new Set([...getSavedFeatureFlags(), flagClean]);
+    const datastr = [...set.keys()].sort().join("\t");
+    window.localStorage.setItem(LOCALSTORAGE_KEY, datastr);
+}
+
+function removeFeatureFlag(flag: string) {
+    const flagLower = flag.trim().toLowerCase();
+    const removed = getSavedFeatureFlags().filter((f) => f !== flagLower);
+    window.localStorage.setItem(LOCALSTORAGE_KEY, removed.join("\t"));
+}
+
+// nothing outside of this file should need anything other than CheckFeatureFlag()...
+// except for tests... so this is how we export but make it CLEAR it's just for testing
+export const _exportForTesting = {
+    LOCALSTORAGE_KEY,
+    getSavedFeatureFlags,
+    addFeatureFlag,
+    removeFeatureFlag,
+};
+
+// PUBLIC functions
+export function CheckFeatureFlag(feature: string): boolean {
```

Comment:
Since it's capitalized, was `CheckFeatureFlag` intended to be a constructor function?

Reply:
Yeah, I'm not sure about CamelCase for our public functions. I'm happy to rename it `checkFeatureFlag()`. 

The non-public functions... could be renamed to start with an `_` ... I've done so many different naming conventions over the decades, I can't keep them straight anymore.